docs/kubernetes/KubernetesIntegration.md

```diff
@@ -0,0 +1,37 @@
+# Integrating the Calico Network Plugin with Kubernetes
+
+This guide will describe the configuration required to use the Calico network plugin in your Kubernetes deployment.
+
+## Setting Up Your Environment 
+   The Calico network plugin looks for three environment variables. If one is not set, it will assume a default value. If you need to override the defaults, you must set these variables in the environment of the _kubelet_ process. 
+
+* #####ETCD_AUTHORITY
+   By default, the Calico network plugin will assume that the ETCD Authority is located at `<MASTER_IP>:6666`. Setting the `ETCD_AUTHORITY` variable in your environment will direct Calico to the correct IP if your cluster is set up differently.
```

Comment:
"ETCD Authority" => "etcd datastore" (the phrase "Authority" isn't a well-known identifier)